packages/upload/src/__tests__/FzUpload.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { describe, it, vi } from "vitest";
+import { mount } from "@vue/test-utils";
+import { FzUpload } from "..";
+
+describe.concurrent("FzUpload", () => {
```

Comment:
amplierei la sezione dei test (preferibilmente non snapshot) con:
- aggiunta di un file
- delete di un file
- test con v-model popolato da N file

Reply:
ho aggiunto due test in d517548
non posso testare l'upload perché non è un browser